tests/run_test.sh

```diff
@@ -36,8 +36,7 @@ export REGRESSIONTEST_LOG=${LOG_DIR}/rt_${TEST_NR}_${TEST_NAME}${RT_SUFFIX}.log
 # Submit the actual test run script
 echo "Test ${TEST_NR} ${TEST_NAME} ${TEST_DESCR}"
 trap 'echo "run_test.sh: Test ${TEST_NAME} killed"; kill $(jobs -p); wait; trap 0; exit' 1 2 3 4 5 6 7 8 10 12 13 15
-./${RUN_SCRIPT} > ${RUNDIR_ROOT}/${TEST_NAME}${RT_SUFFIX}.log 2>&1 &
-wait
+./${RUN_SCRIPT} > ${RUNDIR_ROOT}/${TEST_NAME}${RT_SUFFIX}.log 2>&1
```

Comment:
It's good to have this fixed! We also noticed that RT does not catch the error when generating new baseline. I think this fix will catch that error too.  Does this change impact the parallel run of RT tests?

Reply:
I will have to rely on Dusan to answer this question.

Reply:
I was using parallel rt.sh (using ecFlow) to create baselines and verify and I didn't have any issue with this change.